app/src/main/res/drawable/tab_tray_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/swipe_delete_background_normal_theme" />
```

Comment:
Does this need to be an attribute for dark mode?

Reply:
No, the tabs tray doesn't present private mode, so you just pull the normal theme color for light/dark